packages/ui/src/services/component-config/theme/index.ts

```diff
@@ -0,0 +1,15 @@
+export interface SizePreset<Variables extends string> {
+  variables: Partial<Record<Variables, string>>;
+}
+
+/**
+ * @see To allow union of string and string literals https://stackoverflow.com/questions/61047551/typescript-union-of-string-and-string-literals
+ */
+export type SizeValue<T> = T | (string & {}) | number
```

Comment:
StringWithAutocomplete type helper should be available somewhere in utils/types

Reply:
Thanks for the suggestion, used it here instead